content/rc/security/single-sign-on/saml-sso/saml-integration-okta-generic.md

```diff
@@ -0,0 +1,291 @@
+ 
+    * **Name**
+    * **Description**
+
+    {{<image filename="images/rc/saml/okta_saml_group_2.png" alt="" >}}{{</image>}}
+
+    {{<image filename="images/rc/saml/okta_saml_group_3.png" alt="" >}}{{</image>}}
+
+### Assign your users to the group
+
+1. Click on the group and then click the "Assign people" button.
+
+    {{<image filename="images/rc/saml/okta_saml_group_4.png" alt="" >}}{{</image>}}
+
+2. For each user you wish to add to the group, highlight the user in the table and click on the **"+"** button. You can also use the "Add all" button if needed. Once you have added all the people to your group, click on the **"Save"** button.
+
+    {{<image filename="images/rc/saml/okta_saml_group_5.png" alt="" >}}{{</image>}}
+
+
+### Assign the application to the group
+
+Now that your group is populated with its users, it is time to assign the SAML integration application to your group. From the menu "Applications" -> "Applications" -> "Redis Cloud" :
+
+1. Click on the **Assign to groups** menu item.
+
+    {{<image filename="images/rc/saml/okta_saml_group_6.png" alt="" >}}{{</image>}}
+
+2. In the "Redis Cloud User Group" click on the "Assign" link.
+
+    {{<image filename="images/rc/saml/okta_saml_group_7.png" alt="" >}}{{</image>}}
+
+3. You will now define the redis account mapping string default for this group and click the **"Save and go back"** button. The key-value pair consists of the **lower-cased role name** (ie owner, member, manager, or viewer) AND your **Redis Cloud Account ID** (you can find this information in the upper-right user menu at app.redislabs.com). Finally, click on the **"Done"** button.
+
+    {{<image filename="images/rc/saml/okta_saml_group_8.png" alt="" >}}{{</image>}}
+
+4. The mapping field has now been defined as a default for each member of the group.
+
+    {{<image filename="images/rc/saml/okta_saml_group_9.png" alt="" >}}{{</image>}}
+
+### Editing the mapping field for the group
+
+We might need to modify the Redis mapping field at a certain point in time. It is easy to do so by clicking on the "pencil" icon of the Redis Cloud group in the "Redis Cloud" application screen.
+
+{{<image filename="images/rc/saml/okta_saml_group_10.png" alt="" >}}{{</image>}}
+
+An edit modal screen will appear, where you can modify the mapping field for the whole group.
+
+{{<image filename="images/rc/saml/okta_saml_group_11.png" alt="" >}}{{</image>}}
+
+### Editing the mapping field for a specific user
+
+It is also possible to override the Redis mapping field at an individual user level. In order to do so, click on the "People" menu of the screen, and then on the "pencil" icon of the person you wish to modify the field for.
+
+{{<image filename="images/rc/saml/okta_saml_group_15.png" alt="" >}}{{</image>}}
+
+Next, we need to set the user's "Assignment master" to "Administrator". This will allow us to override the group's policy. We then click on the **"Save"** button.
```

Comment:
```suggestion
Set the user's "Assignment master" to "Administrator". This will allow us to override the group's policy. Select **"Save"**.
```

Reply:
We also generally try to default to 2nd person pronouns instead of "we/us" where possible.